scripts/processPrivateApiData/runtimeContext.ts

```diff
@@ -0,0 +1,32 @@
+import { PrivateApiContextDto, PrivateApiUsageDto } from './dto';
+import { Group, groupedAndDeduped } from './grouping';
+import { joinComponents } from './utils';
+
+/**
+ * Used for determining whether two contexts are equal.
+ */
+export function runtimeContextIdentifier(context: RuntimeContext) {
+  return joinComponents([
+    { key: 'type', value: context.type },
+    { key: 'file', value: context.file ?? 'null' },
+    { key: 'suite', value: context.suite?.join(',') ?? 'null' },
+    { key: 'title', value: context.type === 'definition' ? context.label : context.title },
+    { key: 'helperStack', value: 'helperStack' in context ? context.helperStack.join(',') : 'null' },
```

Comment:
I think we should sort `helperStack` here before joining, since I believe we don't want the order in which we called `addingHelperFunction` to matter here.

Reply:
I’m not sure I understand, but I don't think that sorting it would make any difference to the output. If you have `helperA` which calls `helperB` which calls `helperC`, it's unlikely that there’s going to be, for example, a situation in which `helperB` calls `helperC` which calls `helperA`.